src/build_options.rs

```diff
@@ -29,9 +29,8 @@ pub struct BuildOptions {
         long,
         possible_values = &["2010", "2014", "off"],
         case_insensitive = true,
-        default_value = "2010"
```

Comment:
Please update the doc comment saying something like "The default is the lowest supported value for a target, which is 2010 for x86 and 2014 for arm and powerpc".